CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java

```diff
@@ -301,6 +303,12 @@ public JDBCSourcePane(final JDBCImportController importController) {
         final Label destinationLabel = new Label("Destination:");
         GridPane.setConstraints(destinationLabel, 0, 4, 1, 1, HPos.LEFT, VPos.TOP);
 
+        final Button clearButton = new Button(CLEAR_RESULTS);
+        clearButton.setOnAction((ActionEvent e) -> {
```

Comment:
could you make the lambda parameter final (alternatively in this case you can omit the explicit type altogether i.e. `e ->...`)